pkgs/applications/misc/jgmenu/default.nix

```diff
@@ -22,9 +22,15 @@ stdenv.mkDerivation rec {
     libxml2
     menu-cache
     xorg.libXinerama
+    xorg.libXrandr
+    python3Packages.python
   ];
 
-  makeFlags = [ "prefix=$(out)" ];
+  makeFlags = [ "prefix=${placeholder "out"}" ];
+
+  postPatch = ''
+    sed -i -e "s,export PATH=\$JGMENU_EXEC_DIR:\$PATH,export PATH=\$JGMENU_EXEC_DIR:$out/bin:\$PATH," src/jgmenu_run
```

Comment:
It's way easier to work with `wrapProgram` (you need to add `makeWrapper` to `nativeBuildInputs` for that) here:

```
wrapProgram $out/bin/jgmenu_run \
  --prefix PATH : $extra_path_entry
```

Reply:
Ok.

Notice that `jgmenu` also needs wrapping, otherwise `result/bin/jgmenu init -i` does not work too, failing to find `jgmenu_run`.